### ISSUE TITLE
Increase width of 'narrow' and 'standard' webbook sizes

### ISSUE THREAD

**Issue by colomet** [pressbooks/ideas#340]
### Feature Description

Webbook Width size Standard from 688px to 728px.

Webbook Width size Narrow from 508px to 560px.

### Feature Use Case

Now, the Webbook Width size is 688px. Which is a good size, I guess. But if there is not any special reason why that size is like that, my suggestion is to change the size to 728px. Also, to change Narrow from 508 to 560px.

The reason is if we need to integrate multimedia content, 728px is a standart in the industry of visual complements like ads  and the default youtube size videos are 560px.

The default youtube code will look something like this:
<iframe width=”560″ height=”315″ src=”http://www.youtube.com/embed/XXXXXXXXXX″ frameborder=”0″ allowfullscreen></iframe>

In such way, people can just copy and past code of multimedia elements with not modifications or issues.

**Comment by SteelWagstaff** [comment]
Might be worth revisiting the various reading widths. How often is 'narrow' really used? Should 'wide' be the new default? etc. FYI @MichelleWeremczuk

**Comment by MichelleWeremczuk** [comment]
I agree. What’s the purpose of offering three different reading widths? Do we know what percentage of users chooses each option? Wide reflows well on all screen sizes, so it could work as the new default. These changes are relatively minor—40 pixels for standard view and 52 pixels for narrow view. We could make these changes pretty easily, or we could explore removing views as it isn't clear what these options are offering that wide does not cover already.

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 5y 5m ago with no updates in the last 1y 2m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
